addons/website_slides/views/website_slides.xml

```diff
@@ -462,7 +462,7 @@
                                             tabindex="0"
                                             data-toggle="popover">
                                             <i class="fa fa-thumbs-up fa-1x" role="img" aria-label="Likes" title="Likes"/>
-                                            <t t-esc="slide.likes"/>
+                                            <span class="o_wslides_like_dislike_count" t-esc="slide.likes"/>
```

Comment:
wslides ?

Reply:
According to my secret informant in the website office, the new name convention for classes is something like:
- o (for le Odoo)
- w (for le website)
- slides (for le module)

He gave me the following example: o_wevent_registration_list